.travis.yml

```diff
@@ -9,6 +9,8 @@ jobs:
   include:
     - language: object-c
       osx_image: xcode10.2
+      env:
+        - SKIP_RPC_TESTS=1
```

Comment:
Does this really skip `RPC` tests? Or just local API tests?

Reply:
Yes, it actually skipped the `RPC` test.

Reply:
What I see is you're skipping API tests. CKB RPC here is your API's implementation detail and the tests and usages don't expose the fact that RPC is used. So perhaps `SKIP_API_TESTS` is better.